lib/lintSource.js

```diff
@@ -88,7 +87,7 @@ function lintPostcssResult(
   postcssResult.stylelint.quiet = config.quiet
 
   const newlineMatch = postcssResult.root.toResult().css.match(/\r?\n/)
-  const newline = newlineMatch ? newlineMatch[0] : os.EOL
+  const newline = newlineMatch ? newlineMatch[0] : "\n"
```

Comment:
Holy smokes! You're magician! Thanks! I created `__mocks__/os.js` file, like in documentation, and used `jest.mock("os")` in comment-empty-line-before test. It didn't work.